drake/systems/plants/@Manipulator/Manipulator.m

```diff
@@ -85,24 +86,25 @@
         %   vdot = H\tau
         % but I already have and use Hinv, so use it again here
 
-        xdot = [vToqdot(obj,q)*v; vdot];
+        kinsol = obj.doKinematics(q);
+        xdot = [vToqdot(obj,kinsol)*v; vdot];
```

Comment:
looks like no.  but it probably should (it calls doKinematics immediately).  right @tkoolen ?